src/XRP/network-clients/json-rpc-network-client.ts

```diff
@@ -81,4 +83,34 @@ export default class JsonRpcNetworkClient {
     const accountLinesResponse: AccountLinesResponse = axiosResponse.data
     return accountLinesResponse
   }
+
+  /**
+   * Submits a gateway_balances request to the rippled JSON RPC.
+   * @see https://xrpl.org/gateway_balances.html
+   *
+   * @param account The XRPL account for which to retrieve balances.
+   * @param hotwallet (Optional) An operational address to exclude from the balances issued, or an array of such addresses.
+   * @see https://xrpl.org/issuing-and-operational-addresses.html
+   */
+  public async getGatewayBalances(
+    account: string,
+    hotwallet?: string | Array<string>,
+  ): Promise<GatewayBalancesResponse> {
+    const gatewayBalancesRequest = {
+      method: 'gateway_balances',
+      params: [
+        {
+          account: account,
+          hotwallet: hotwallet,
```

Comment:
I believe this can also be written as just `hotwallet,`

Reply:
yee, and `account` as `account`

https://alligator.io/js/object-property-shorthand-es6/

Reply:
someday this'll sink in